src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's contact in the address book.
+ */
+public class Contact {
+	
+    public String value;
```

Comment:
I assumed the creator would like it to be public as seen in the children class thats why I left it as public.